ansible/roles/invoker/tasks/deploy.yml

```diff
@@ -63,16 +63,38 @@
         --name invoker{{ groups['invokers'].index(inventory_hostname) }}
         --hostname invoker{{ groups['invokers'].index(inventory_hostname) }}
         --restart {{ docker.restart.policy }}
+        -e JAVA_OPTS=-Xmx{{ invoker.heap }}
+        -e INVOKER_OPTS='{{ invoker.arguments }}'
         -e COMPONENT_NAME=invoker{{ groups['invokers'].index(inventory_hostname) }}
-        -e CONSULSERVER_HOST={{ groups['consul_servers'] | first }}
-        -e CONSUL_HOST_PORT4={{ consul.port.http }}
         -e PORT=8080
+        -e KAFKA_HOST={{ groups['kafka']|first }}
+        -e KAFKA_HOST_PORT={{ kafka.port }}
+        -e DB_PROTOCOL={{ db_protocol }}
+        -e DB_PROVIDER={{ db_provider }}
+        -e DB_HOST={{ db_host }}
+        -e DB_PORT={{ db_port }}
+        -e DB_USERNAME={{ db_username }}
+        -e DB_PASSWORD={{ db_password }}
+        -e DB_WHISK_ACTIONS={{ db.whisk.actions }}
+        -e DB_WHISK_ACTIVATIONS={{ db.whisk.activations }}
+        -e WHISK_API_HOST_PROTO={{ whisk_api_host_proto | default('https') }}
+        -e WHISK_API_HOST_PORT={{ whisk_api_host_port | default('443') }}
+        -e WHISK_API_HOST_NAME={{ whisk_api_host_name | default(groups['edge'] | first) }}
+        -e RUNTIMES_MANIFEST='{{ runtimesManifest | to_json }}'
         -e SELF_DOCKER_ENDPOINT=localhost
-        -e SERVICE_CHECK_HTTP=/ping
-        -e SERVICE_CHECK_TIMEOUT=2s
-        -e SERVICE_CHECK_INTERVAL=15s
-        -e JAVA_OPTS=-Xmx{{ invoker.heap }}
-        -e INVOKER_OPTS='{{ invoker.arguments }}'
+        -e DOCKER_REGISTRY={{ docker_registry }}
+        -e DOCKER_IMAGE_PREFIX={{ docker_image_prefix }}
+        -e DOCKER_IMAGE_TAG={{ docker_image_tag }}
+        -e INVOKER_CONTAINER_NETWORK=bridge
```

Comment:
I don't think this was hardcoded to "bridge" before, was it? This needs to be configurable.